screeps-game-api/src/objects/impls/creep.rs

```diff
@@ -24,6 +24,10 @@ impl Creep {
         js_unwrap!(@{self.as_ref()}.carry[RESOURCE_ENERGY])
     }
 
+    pub fn owner_name(&self) -> String {
```

Comment:
It'd be nice to match this naming with `OwnedStructureProperties::owner` and `ConstructionSite::owner`. I think I might just change those two, though, since `owner_name` is strictly more accurate.

Reply:
It might be nice, but I'm not too concerned with typing everything that's a string value underneath.

My main concern is that while things like `Username` or `ObjectId` could be useful within one instance, any screeps AI will need to serialize them and retrieve them from memory every tick, and at that point there's no more safety than just treating them as strings anyways.

I've opened https://github.com/daboross/screeps-in-rust-via-wasm/issues/29.